openstack/clustering/v1/policies/requests.go

```diff
@@ -54,3 +58,63 @@ func List(client *gophercloud.ServiceClient, opts ListOptsBuilder) pagination.Pa
 		return p
 	})
 }
+
+// CreateOpts params
+type CreateOpts struct {
+	Name string `json:"name"`
+	Spec Spec   `json:"spec"`
+}
+
+type Spec struct {
```

Comment:
Even though `Spec` is used in requests, because it's used in responses _and_ has a custom unmarshaller, it should go in `results.go`. That's more of an internal organization style: as soon as it's part of a result, it goes in `results.go`.

Reply:
Done